src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -45,6 +41,38 @@
     <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj">
       <TargetGroup>netcore50</TargetGroup>
     </ProjectReference>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+    <!-- We need to add this for now because these three types exist in System.Private.Interop.dll and Windows.winmd. -->
+    <SeedTypePreference Include="Windows.Foundation.Point">
+      <Assembly>System.Private.Interop</Assembly>
+    </SeedTypePreference>
+    <SeedTypePreference Include="Windows.Foundation.Rect">
+      <Assembly>System.Private.Interop</Assembly>
+    </SeedTypePreference>
+    <SeedTypePreference Include="Windows.Foundation.Size">
+      <Assembly>System.Private.Interop</Assembly>
+    </SeedTypePreference>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+    <TargetingPackReference Include="System.Private.Interop" />
+    <TargetingPackReference Include="System.Private.Corelib" />
+    <TargetingPackReference Include="System.Private.Threading" />
```

Comment:
System.Runtime.WindowRuntime need to use Progress<T> in System.Private.Threading